.github/workflows/release-binary.yml

```diff
@@ -63,12 +67,8 @@ jobs:
         with:
           version: latest
           args: release -f .goreleaser/linux-arm64.yml --clean
-          workdir: /
-        env: 
+        env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-          SLACK_WEBHOOK: "${{ secrets.RELEASE_SLACK_WEBHOOK }}"
-          DISCORD_WEBHOOK_ID: "${{ secrets.DISCORD_WEBHOOK_ID }}"
-          DISCORD_WEBHOOK_TOKEN: "${{ secrets.DISCORD_WEBHOOK_TOKEN }}"
```

Comment:
In linux-arm64.yml, slack and discord integration are disabled.

https://github.com/projectdiscovery/naabu/blob/a687d975c0bb7791bef01ccdfc2cdd6962bc062b/.goreleaser/linux-arm64.yml#L28-L37